tests/testthat/test-derive_param_tte.R

```diff
@@ -633,3 +633,14 @@ test_that("derive_param_tte Test 9: error is issued if parameter code already ex
     regexp = "^The parameter code 'TTAE' does already exist in `dataset`.$"
   )
 })
+
+## Test 10: error is issued if package does not exist ----
+test_that("derive_param_tte Test 10: error is issued if package does not exist", {
+
+  expected_output <- list_tte_source_objects(package = "admiral")
+
+  expect_error(
+    list_tte_source_objects(package = "tte"),
+    regexp = "No package called 'tte' is installed and hence no `tte_source` objects are available"
+  )
```

Comment:
If you check for a warning or error there's no need for an "expected" output apart from the `regexp` you define inside `expect_error()`.

```suggestion
  expect_error(
    list_tte_source_objects(package = "tte"),
    regexp = "No package called 'tte' is installed and hence no `tte_source` objects are available"
  )
```